### ISSUE TITLE
[WCAG] Project Detail Pages with lighter background colors trigger contrast error

### ISSUE THREAD

**Issue by arielkidwell** [vigetlabs/pointless-corp-astro#23]
**Examples**
- https://pointlesscorp.netlify.app/storyboard/
- https://pointlesscorp.netlify.app/baby-bookie/
- https://pointlesscorp.netlify.app/jambells/ (this one surprised me)
- https://pointlesscorp.netlify.app/gifbowl/
- https://pointlesscorp.netlify.app/boom-carded/
- https://pointlesscorp.netlify.app/graba/
- https://pointlesscorp.netlify.app/scurry/
- https://pointlesscorp.netlify.app/glimpsee/

**Comment by arielkidwell** [comment]
@ten1seven Here's some further documentation on this - it's lengthy but I went through each one.

https://pointlesscorp.netlify.app/storyboard/
- Top passes, bottom doesn't
- Have to go way darker to pass
![Image](https://github.com/user-attachments/assets/c48042b0-a14b-4cfa-9fa2-e9599b54b998)

https://pointlesscorp.netlify.app/baby-bookie/
- Have to go way darker to pass AND it conflicts with green logo graphic

https://pointlesscorp.netlify.app/starbnb/
- Had to go slightly lighter on link color which passes but makes it out of sync with. logo

https://pointlesscorp.netlify.app/gifbowl/
- Tried darkening background but then darker text got flagged

https://pointlesscorp.netlify.app/boom-carded/
- Accessibility nightmare

https://pointlesscorp.netlify.app/tracktor/
- Needs color rethink, if I pull one lever, another two break

https://pointlesscorp.netlify.app/jinglebots/
- Needs color rethink, if I pull one lever, another two break

https://pointlesscorp.netlify.app/scurry/
- Darker background works pretty well but needs updated logo

https://pointlesscorp.netlify.app/glimpsee/
- The pinkish text still doesn't meet it but not sure what other direction to go but white

https://pointlesscorp.netlify.app/the-data-driven-gardener/
- Needs color rethink, if I pull one lever, another two break (lots of green levels)

**Comment by ten1seven** [comment]
@arielkidwell Good catch, I should have updated the footer to use the same black/white contrast function I used for the header. I'll make sure to do that.

On the theme colors, since this is a rebuild of the existing site and we're doing a bunch to improve the accessibility of the non-project page elements (homepage, nav, and footer), I'm inclined not to let the color selections of individual Pointless teams block launching this rebuild. I recommend turning this inventory into a new ticket to have a designer review for color adjustment recommendations post-launch. @carolynhack, what do you think?

**Comment by ten1seven** [comment]
@arielkidwell footer updates are now on production!

![CleanShot 2024-11-25 at 09 54 52@2x](https://github.com/user-attachments/assets/841d53ec-2b12-4ba8-8592-5ca8bf999bbd)

**Comment by arielkidwell** [comment]
@ten1seven Updated footer working great! I'll reassign to @carolynhack for her preference on how to move forward with [color contrast issues.](https://github.com/vigetlabs/pointless-corp-astro/issues/23#issuecomment-2498379130
)

**Comment by carolynhack** [comment]
@ten1seven @arielkidwell I don't think color contrast on individual pages should hold up a relaunch, especially if it's just 1:1 duplicating what's already live on the current site. 

I think Jeremy's recommendation to open a new ticket with all of the contrast issues to be addressed independent of the launch is a good idea. I can't imagine it would take a designer more than an hour to make recommendations on the 10 links above.

**Comment by arielkidwell** [comment]
Opened Issue #27 to address.
